shared/output/cloud/compute_small/lib/google/cloud/compute/v1/addresses/rest/client.rb

```diff
@@ -0,0 +1,516 @@
+              #
+              #   @param filter [::String]
+              #     A filter expression that filters resources listed in the response. The expression must specify the field name, a comparison operator, and the value that you want to use for filtering. The value must be a string, a number, or a boolean. The comparison operator must be either =, !=, >, or <.
+              #
+              #     For example, if you are filtering Compute Engine instances, you can exclude instances named example-instance by specifying name != example-instance.
+              #
+              #     You can also filter nested fields. For example, you could specify scheduling.automaticRestart = false to include instances only if they are not scheduled for automatic restarts. You can use filtering on nested fields to filter based on resource labels.
+              #
+              #     To filter on multiple expressions, provide each separate expression within parentheses. For example, (scheduling.automaticRestart = true) (cpuPlatform = "Intel Skylake"). By default, each expression is an AND expression. However, you can include AND and OR expressions explicitly. For example, (cpuPlatform = "Intel Skylake") OR (cpuPlatform = "Intel Broadwell") AND (scheduling.automaticRestart = true).
+              #   @param max_results [::Integer]
+              #     The maximum number of results per page that should be returned. If the number of available results is larger than maxResults, Compute Engine returns a nextPageToken that can be used to get the next page of results in subsequent list requests. Acceptable values are 0 to 500, inclusive. (Default: 500)
+              #   @param order_by [::String]
+              #     Sorts list results by a certain order. By default, results are returned in alphanumerical order based on the resource name.
+              #
+              #     You can also sort results in descending order based on the creation timestamp using orderBy="creationTimestamp desc". This sorts results based on the creationTimestamp field in reverse chronological order (newest result first). Use this to sort resources like operations so that the newest operation is returned first.
+              #
+              #     Currently, only sorting by name or creationTimestamp desc is supported.
+              #   @param page_token [::String]
+              #     Specifies a page token to use. Set pageToken to the nextPageToken returned by a previous list request to get the next page of results.
+              #   @param project [::String]
+              #     Project ID for this request.
+              #   @param region [::String]
+              #     Name of the region for this request.
+              #
+              # @yield [result, env] Access the result along with the Faraday environment object
+              # @yieldparam result [::Google::Cloud::Compute::V1::AddressList]
+              # @yieldparam env [::Faraday::Env]
```

Comment:
I think you're actually yielding the Faraday::Response rather than the Faraday::Env. Can you double-check? And maybe determine which interface we want to expose? (I think the point of the block form is to provide access to the low-level response data such as response headers, for the uncommon use cases that care about it, so maybe decide based on completeness of the data that can be accessed?)

Also, can you document whether or not the block is called in the case of an error response? I think it should be called, but I'm not sure what the implementation currently does.

Reply:
this whole yielding thing is in progress

Reply:
yielding has been changed (the docs might be still stale?)